src/Cassandra/Serialization/Graph/GraphSON2/Dse/Duration2Serializer.cs

```diff
@@ -0,0 +1,42 @@
+﻿//
+//       Copyright (C) DataStax Inc.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+using Cassandra.Serialization.Graph.Tinkerpop.Structure.IO.GraphSON;
+
+namespace Cassandra.Serialization.Graph.GraphSON2.Dse
+{
+    internal class Duration2Serializer : StringBasedSerializer
```

Comment:
`Duration2` ?

Reply:
I used the same naming strategy that the Tinkerpop serializes use to differentiate serializers for the same type but different GraphSON versions. This is the `Duration` serializer for GraphSON2

Reply:
Sounds good, nvm!